docs/source/deployment_client.rst

```diff
@@ -0,0 +1,119 @@
+.. _mlflow_deployment_client:
+
+MLflow Deployment Client
+=========================
+
+MLflow Deployment Client is a set of CRUD and query APIs to interact models that are served across various platforms.
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 1
+
+Overview
+--------
+
+The MLflow Deployment Client provides unified CRUD and query APIs to interact with models that are served across various platforms. MLflow provides several built-in deployment clients that provide a consistent and platform-agnostic interface to interact with models served on different platforms, as well as a interface that can be used to implement custom deployment clients via plugins.
+
+
+Builtin Deployment Clients
+--------------------------
+
+The MLflow library provides several built-in deployment clients that can be used to interact with models served on different platforms. The following are the built-in deployment clients provided by MLflow:
+
+* :py:class:`mlflow.deployments.MlflowDeploymentClient`: A deployment client for served models in the MLflow Deployments Server
+* :py:class:`mlflow.deployments.DatabricksDeploymentClient`: A deployment client for served models on Databricks.
+* :py:class:`mlflow.deployments.OpenAIDeploymentClient`: A deployment client for served models on OpenAI or Azure OpenAI platforms.
+
+Deployment Client Factory functions
```

Comment:
Might want to rename this header to not use the SWE term "Factory" as the target audience might be very confused by a design pattern reference